evaluation.py

```diff
@@ -0,0 +1,101 @@
+from submodules.HeatMap import HeatMap # https://github.com/LinShanify/HeatMap
+from constants import *
+# from util import *
+
+import matplotlib.pyplot as plt
+import numpy as np
+from PIL import Image
+import cv2
+
+class Evaluation():
+
+    def __init__(self, model, weights, df, num_hg_blocks, batch_size=1):
+        self.model = model              # json of model to be evaluated
+        self.weights = weights          # weights of model to be evaluated
+        self.df = df                    # df of the the annotations we want
+        self.num_hg_blocks = num_hg_blocks
+        self.batch_size = batch_size
+
+    # https://stackoverflow.com/questions/30227466/combine-several-images-horizontally-with-python
+    def hstack_images(self, images, filename):
+        widths, heights = zip(*(i.size for i in images))
+
+        total_width = sum(widths)
+        max_height = max(heights)
+
+        new_im = Image.new('RGB', (total_width, max_height))
+
+        x_offset = 0
+        for im in images:
+            new_im.paste(im, (x_offset,0))
+            x_offset += im.size[0]
+
+        new_im.save(filename)
+
+    # https://www.geeksforgeeks.org/concatenate-images-using-opencv-in-python/
+    def vstack_images(self, img_list, interpolation=cv2.INTER_CUBIC):
```

Comment:
**Nitpick**
@nicolepeverley I think you mentioned this is a private method so you could change the name to `__vstack_images`.